selfdrive/car/gm/values.py

```diff
@@ -12,6 +12,12 @@ class CruiseButtons:
   MAIN        = 5
   CANCEL      = 6
 
+class AccState:
+  OFF        = 0
+  ACTIVE     = 1
+  FAULTED    = 3
```

Comment:
@vntarasov , have you seen the AccState becoming FAULTED for other reasons other than the low speed lockout?

Reply:
Good point, let me find someone willing to test it on 2018 Volt. On my 2017 Volt, speed-related FAULTED clears out as soon as openpilot disengages, but I haven't tested it on 2018 models. It would be rather inconvenient if PCM stays faulted.

Re: other ways to get to FAULTED: There was a bug with "isActive" vs "isEnabled", where PCM would go to FAULTED (for a few seconds on 2017 Volts, and until ACC is reset on 2018), fixed in https://github.com/commaai/openpilot/pull/352. I don't know of other ways to get it to FAULTED other than stop sending keepalive messages (same GasRegenCmd, with "enabled" = False).